### PR TITLE
CDK: fixes for destination-postgres

### PR DESCRIPTION
**Pull request by gisripa**

### CODE REVIEW

File: airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/db/jdbc/JdbcUtils.kt [hunk 1]
```diff
@@ -62,7 +62,7 @@ object JdbcUtils {
             JDBCType.VARCHAR,
             JDBCType.LONGVARCHAR
         )
-    @JvmField val defaultSourceOperations: JdbcSourceOperations = JdbcSourceOperations()
+    @JvmStatic val defaultSourceOperations: JdbcSourceOperations = JdbcSourceOperations()
```
Comment (gisripa) [inline airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/db/jdbc/JdbcUtils.kt:65]: @jdpgrailsdev  This was accessed as a static member in Java, any better way to do than the escape hatch ?

Comment (stephane-airbyte) [reply airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/db/jdbc/JdbcUtils.kt:65]: not sure what the escape hatch is

File: airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/integrations/base/AirbyteExceptionHandler.kt [hunk 2]
```diff
@@ -154,8 +154,7 @@ class AirbyteExceptionHandler : Thread.UncaughtExceptionHandler {
             }
         }
 
-        @VisibleForTesting
-        fun addCommonStringsToDeinterpolate() {
+        internal fun addCommonStringsToDeinterpolate() {
```
Comment (stephane-airbyte) [inline airbyte-cdk/java/airbyte-cdk/core/src/main/kotlin/io/airbyte/cdk/integrations/base/AirbyteExceptionHandler.kt:157]: yes

File: airbyte-cdk/java/airbyte-cdk/core/src/testFixtures/kotlin/io/airbyte/cdk/testutils/TestDatabase.kt [hunk 3]
```diff
@@ -300,18 +298,21 @@ protected constructor(@JvmField val container: C) : AutoCloseable {
         }
 
         fun withSsl(sslMode: MutableMap<Any?, Any?>?): B {
-            return with(JdbcUtils.SSL_KEY, true)!!.with(JdbcUtils.SSL_MODE_KEY, sslMode)
+            return with(JdbcUtils.SSL_KEY, true).with(JdbcUtils.SSL_MODE_KEY, sslMode)
         }
 
         companion object {
-            val DEFAULT_CDC_REPLICATION_INITIAL_WAIT: Duration? = Duration.ofSeconds(5)
+            private val DEFAULT_CDC_REPLICATION_INITIAL_WAIT: Duration = Duration.ofSeconds(5)
```
Comment (stephane-airbyte) [inline airbyte-cdk/java/airbyte-cdk/core/src/testFixtures/kotlin/io/airbyte/cdk/testutils/TestDatabase.kt:305]:
```
@JvmStatic protected val DEFXXX: Duration = XXX
   private set
```

Comment (gisripa) [reply airbyte-cdk/java/airbyte-cdk/core/src/testFixtures/kotlin/io/airbyte/cdk/testutils/TestDatabase.kt:305]: in java it looks like this  `getDEFAULT_CDC_REPLICATION_INITIAL_WAIT()` by using above convention. Not sure why. its part of nested ConfigBuilder class's companion not the TestDatabase

Comment (stephane-airbyte) [reply airbyte-cdk/java/airbyte-cdk/core/src/testFixtures/kotlin/io/airbyte/cdk/testutils/TestDatabase.kt:305]: what about @JvmField? That should get rid of the constructor, no?

**Review by stephane-airbyte — APPROVED** [review]

**Pull request merged**